build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
What is this script for? It doesn’t seem to be used/referenced anywhere

Reply:
It's for building and uploading the package, makes my life easier.